recipes/devices/cuboxp.sh

```diff
@@ -37,7 +38,7 @@ INIT_TYPE="init.nextarm" # init.{x86/nextarm/nextarm_tvbox}
 # Modules that will be added to intramsfs
 MODULES=("overlay" "squashfs" "nls_cp437")
 # Packages that will be installed
-PACKAGES=("u-boot-tools" "device-tree-compiler")
+# PACKAGES=("u-boot-tools" "device-tree-compiler")
```

Comment:
I didn't see `dtc` being used anywhere, but please confirm if it is needed. 
It might be prudent to add both `u-boot-tools` and `device-tree-compiler` to the base recipes - what do you think?

Reply:
i have used both for testing, but could easily do apt-get in the rare occasions.